proto/src/lib.rs

```diff
@@ -218,3 +205,19 @@ pub mod ibc {
 pub mod ics23 {
     include_proto!("ics23.rs");
 }
+
+#[cfg(feature = "std")]
```

Comment:
In my view, there is no use case where this crate does not need to be compatible with Go, so the ``base64`` encoding/decoding should not be optional.

I don't know what use case you guys have for this crate to support ``no_std``, but you can try to explore [``encode_config_slice``](https://docs.rs/base64/latest/base64/fn.encode_config_slice.html) and [``decode_config_slice``](https://docs.rs/base64/latest/base64/fn.decode_config_slice.html) with ``default-features=false`` for [``base64``](https://docs.rs/crate/base64/latest/source/Cargo.toml) crate

Reply:
https://github.com/informalsystems/ibc-rs/pull/1914/commits/245c1e761d2d47c25a1327976f775e046119f1bb